front-page.php

```diff
@@ -451,20 +459,38 @@
     $press_pub = $press_meta['_igv_press_publication'][0];
     $press_url = $press_meta['_igv_press_url'][0];
```

Comment:
because this is being set from an array index not getting as a single value from `get_post_meta` then next conditional `if (!empty($press_url)) {` isnt the issue with the php warning. The code only works if the user fills the value in every time